report/src/system_evaluation/results.tex

```diff
@@ -32,28 +32,40 @@ \section{System evaluation}\label{sec:experiment_results}
+    4                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{9/10} & 10/10                  \\ \hline
+    \end{tabular}}
+    \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each $k$-value}
+    \label{lst:resultsTablet}
+\end{table}
 
-To create an environment that more accurately represents a meeting room in use, several devices were used for testing. 
-The method for collecting results from these tests mirrored previous ones, by making $10$ classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 
-The classification results were gathered in the locations shown on Figure \ref{fig:experiment_room}
-During the experiment, k-values of $2$, $3$, and $4$ were used.  
-The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.
-Table \ref{lst:resultsPhone_precision} summarizes the experiment's results, indicating the percentage of true positive classifications. 
+On Table \ref{lst:resultsPhone_precision} see that a lower $k$-value results in a larger uncertainty in the classification.
+However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_setup}) a large uncertainty in classification is observed.
+We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
+This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.
+
+Similarly, for all devices, when performing the classification in locations H, I, J, K, and L shows, the results are vastly different from the other locations.
+These locations are all outside of the room. 
+We suspect that this can be due to two things.
+First and foremost, it might be due to the created map being unreliable. 
+When creating the map, the wall separating the room and the common area might cause large fluctuations in the strength of the received signal. 
+Therefore, classification using the map becomes unreliable.
+We tried to accomodate this by enforcing all map entries to have a standard deviation of 6.5 \todo{units and 6.5}. 
```

Comment:
What was that todo about???

Also: explain why sd of 6.5 was chosen in the first place.

```suggestion
We tried to accomodate this by enforcing all map entries to have a standard deviation of $6.5$dBm. 
```